service/src/test/java/bio/terra/workspace/service/workspace/WorkspaceServiceTest.java

```diff
@@ -122,6 +125,9 @@ class WorkspaceServiceTest extends BaseConnectedTest {
   // application-app-test.yml file.
   private static final String TEST_WSM_APP = "TestWsmApp";
 
+  private static final String PARENT_FOLDER_NAME = "ParentFolderName";
+  private static final String FOLDER_NAME = "FolderName";
```

Comment:
not used? 

since there's just one folder, keep folder_name and remove parent_folder_name.